src/en/infernalvoidscans/src/eu/kanade/tachiyomi/extension/en/infernalvoidscans/HiveScans.kt

```diff
@@ -20,6 +22,14 @@ class HiveScans : Iken(
         }
         .build()
 
+    val PageRegex = Regex("""https://storage.\w+.\w+/public//upload/series/(\w+)/(\w+)/(\d+).webp""")
```

Comment:
Is the double slash intentional? Is  that something that should be optional in case they fix it?

Unrelated, but we should really have some kind of helper functions to parse next.js' [RSC](https://github.com/ScriptedAlchemy/rsnext/blob/8f5f0ef141a907d083eedb7c7aca52b04f9d258b/packages/next/src/server/app-render/use-flight-response.tsx#L10).

Reply:
Also regex will fail if the slug of the series contains special characters